OmniKitUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -282,6 +319,17 @@ class OmnipodSettingsViewController: RileyLinkSettingsViewController {
             switch actions[indexPath.row] {
             case .suspendResume:
                 return suspendResumeTableViewCell
+            case .testCommand:
+                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
+                cell.textLabel?.text = LocalizedString("Test Command", comment: "The title of the command to run the test command")
+                cell.accessoryType = .disclosureIndicator
+                return cell
+            case .checkBeeps:
+                let cell = tableView.dequeueReusableCell(withIdentifier: SettingsTableViewCell.className, for: indexPath)
+                cell.textLabel?.text = LocalizedString("Check Beeps", comment: "The title of the command to run check beeps")
```

Comment:
"Check Beeps" is somewhat ambiguous. My first assumption was that it reported the status of whether beeps were enabled. This should indicate it will play a sound on the pod.  Maybe "Emit Pod Beep", or "Play Test Beep"

Reply:
Check Beeps renamed to Play Test Beeps